orderbook/openapi.yml

```diff
@@ -384,6 +384,8 @@ components:
           $ref: "#/components/schemas/Signature"
         signingScheme:
           $ref: "#/components/schemas/SigningScheme"
+        from:
+          $ref: "#/components/schemas/Address"
```

Comment:
nanonit: maybe add nullable (to me this makes it more obvious that the value can be omitted)?